### ISSUE TITLE
ReduceSumSquare op much slower than a combination of Mul and ReduceSum.

### ISSUE THREAD

**Issue by prabhat00155** [microsoft/onnxruntime#3236]
**Describe the bug**
ReduceSumSquare op seems really slow. I replaced it with a combination of Mul and ReduceSum op and they seem to be much faster. I ran profiling on the onnx model, which shows ReduceSumSquare to be the slowest op by far.
```
duration | op name
21682341 | ReduceSumSquare
```
After replacing it with Mul and ReduceSum:
```
duration | op name
3497375 | Mul
1311515 | ReduceSum
```
I used onnxruntime's SessionOptions to enable profiling.
```
sess_options = onnxruntime.SessionOptions()
sess_options.enable_profiling = True
sess = onnxruntime.InferenceSession(model, sess_options=sess_options)
...
prof_file = sess.end_profiling()
```
**Urgency**

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Darwin Kernel Version 19.3.0
- ONNX Runtime installed from (source or binary): binary
- ONNX Runtime version: 1.2.0
- Python version: 3.6.8
- Visual Studio version (if applicable):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

**To Reproduce**
Describe steps/code to reproduce the behavior:

**Expected behavior**

**Screenshots**

**Additional context**
Add any other context about the problem here. If the issue is about a particular model, please share the model details as well to facilitate debugging.

**Comment by skottmckay** [comment]
Can you share the two models (one with ReduceSumSquare and one with Mul+ReduceSum) and the input used so all the attributes/inputs for the nodes are known and they can be checked in a performance test?

**Comment by prabhat00155** [comment]
Here is a sample input(array of floats of size 10):
```
array([[-0.5045926 ,  0.3324247 , -0.53061676, -0.94384676, -0.68722814,
        -0.7007891 ,  0.5135556 , -1.7547697 , -1.494595  ,  0.12554839]],
      dtype=float32)
```
Emailed you the models.

**Comment by skottmckay** [comment]
Should be fixed with #3265
